nav2_map_server/src/map_server.cpp

```diff
@@ -35,6 +35,8 @@ MapServer::MapServer()
 
   // Declare the node parameters
   declare_parameter("yaml_filename", rclcpp::ParameterValue(std::string("map.yaml")));
+
+  RCLCPP_WARN(get_logger(), "map_server lifecycle node launched. Awaiting lifecycle transitions");
```

Comment:
I think that should really be an `INFO` or `DEBUG`.

If anything, this should be added to the lifecycle node wrapper we have so it covers them all.

Reply:
> I think that should really be an INFO or DEBUG.

Yeah, I agree in principle, but I really want it to stand out in the case they `ros2 run` a node, before they file an issue.

> If anything, this should be added to the lifecycle node wrapper we have so it covers them all.

Again, I agree in principle, but we don't use the wrapper everywhere. We use bare lifecycle nodes in some places. And I really want it to stand out in case they `ros2 run` a node. By putting it at the end of the derived class constructor, I can ensure it is the last line on the screen, helping it to stand out more.

We've had *at least* 3 or 4 people who've run into this issue and a few of them have filed issues. I feel like this needs to really standout until people are used to lifecycle nodes.

Reply:
Well, right now the "WARN"  and "INFO" don't color differently in the terminal, so I think its not that much of a eye-catching item.

This isn't really a warning as much as an FYI. If anything, I think it should be an info with some more context like:

"Lifecycle node has been launched. Waiting on external lifecycle transitions to activate... See https://design.ros2.org/articles/node_lifecycle.html for more information."

Reply:
> Well, right now the "WARN" and "INFO" don't color differently in the terminal, so I think its not that much of a eye-catching item.

I thought that was the case as well, but it did in fact color for me when I was testing today. I thought that was something you had pushed up-stream actually.

> "Lifecycle node has been launched. Waiting on external lifecycle transitions to activate... See https://design.ros2.org/articles/node_lifecycle.html for more information."

I agree. More context would be better.

Maybe what I can do is make the messages an RCLCPP_INFO, and make it stand out more with distinctive formatting. I'll experiment with that.

Reply:
I think an INFO with a "hey, look at this link if you don't know what's up" is an option. I'm not sure how you want to change the formatting, but that also sounds fine. I just wouldn't want this to be an elevated log. I used to track metrics like numbers of warnings per run and such and this isn't really a warning or error, its an FYI. I really belongs as a DEBUG but I understand almost no one would see it and still file tickets. INFO seems to be the sweet spot to me.